tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/ComplianceRequirement.scala

```diff
@@ -43,8 +43,8 @@ object ComplianceRequirement {
   }
 
   def mergeFromManifests(
-      manifests: Traversable[JSDependencyManifest]
-  ): Traversable[ComplianceRequirement] = {
+      manifests: Iterable[JSDependencyManifest]
+  ): Iterable[ComplianceRequirement] = {
```

Comment:
That is not an acceptable change, because it breaks a public API.

You need to use a `type` alias in an `Compat` object instead.